Makefile

```diff
@@ -1,19 +1,27 @@
+CONTAINER_NAME=packit
+CONTAINER_EXEC=podman exec $(CONTAINER_NAME)
+
 prepare-check:
 	ansible-playbook -b -K -i inventory-local -c local ./recipe-tests.yaml
 
-check:
-	tox
+container:
+	rsync -a $(CURDIR)/ /tmp/packit
+	podman container inspect packit || podman run --name $(CONTAINER_NAME) -ti -d -v /tmp/packit:/src:Z -w /src fedora:30
+	sleep 2
+	$(CONTAINER_EXEC) bash -c "dnf -y install make"
+
+container_kill:
+	podman stop $(CONTAINER_NAME) || true
+	podman rm -f $(CONTAINER_NAME)
```

Comment:
Shouldn't the `container_kill` be also called somewhere?
Maybe in `check_in_container`?

Reply:
Theoretically no. It is there more less just for local debugging. Inside CI you will everybody have clean env. So that destroying is not important

Reply:
Well, I meant: what If I don't want to have the container running after I run the tests locally?